decompiler/ff7_field/ff7_field_engine.cpp

```diff
@@ -1600,6 +1880,69 @@ void FF7::FF7CameraInstruction::processInst(Function& func, ValueStack&, Engine*
     }
 }
 
+void FF7::FF7CameraInstruction::processNFADE(CodeGenerator* codeGen)
```

Comment:
If each of these "group" instructions just has a switch case to a dedicated method then might as well create a class for each one instead?